### DISCUSSION TITLE
How to disable boot and kernel log output on HDMI out

### DISCUSSION THREAD

**Discussion by slip76** [OE4T/meta-tegra#790] [Q&A] [answered]
The device is based on Jetson AGX Xavier. I need to hide any stuff that can help to identify the OS for kiosk device. I've already replaced nvidia logo with the custom one. Now trying to get rid of _any_ messages on the screen during boot.
Found a bit of information but it is for nvidia SDK or for different platform. I've tried the following but this does not help:
```
repos/meta-tegra$ git diff
diff --git a/conf/machine/include/tegra-common.inc b/conf/machine/include/tegra-common.inc
index e8f7e7d0..492ec4f8 100644
--- a/conf/machine/include/tegra-common.inc
+++ b/conf/machine/include/tegra-common.inc
@@ -10,7 +10,7 @@ UBOOT_EXTLINUX ?= "1"
 UBOOT_EXTLINUX_FDTDIR ?= ""
 UBOOT_EXTLINUX_ROOT ?= "${cbootargs} ${KERNEL_ROOTSPEC}"
 # Console setting comes from ${cbootargs}
-UBOOT_EXTLINUX_CONSOLE = ""
+UBOOT_EXTLINUX_CONSOLE = "console=tty1"
 UBOOT_EXTLINUX_KERNEL_ARGS ?= "${bootargs}"
 UBOOT_EXTLINUX_INITRD ?= "${@'../initrd' if d.getVar('INITRAMFS_IMAGE') != '' and d.getVar('TEGRA_INITRAMFS_INITRD') == '1' else ''}"

diff --git a/conf/machine/jetson-xavier.conf b/conf/machine/jetson-xavier.conf
index a51874cd..416b3370 100644
--- a/conf/machine/jetson-xavier.conf
+++ b/conf/machine/jetson-xavier.conf
@@ -5,7 +5,7 @@
 require conf/machine/include/tegra194.inc

 KERNEL_DEVICETREE ?= "_ddot_/_ddot_/_ddot_/_ddot_/nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-p2822-0000.dtb"
-KERNEL_ARGS ?= "console=ttyTCU0,115200 console=tty0 fbcon=map:0"
+KERNEL_ARGS ?= "console=tty1 fbcon=map:0"

 EMMC_SIZE ?= "31276924928"
 EMMC_DEVSECT_SIZE ?= "512"
```

**Comment by madisongh** [comment, marked as answer]
Try `KERNEL_ARGS ?= "console=ttyTCU0,115200 fbcon=map:0 quiet"`.  That will direct the kernel messages to the serial console, rather than the video display; adding the `quiet` parameter reduces the amount of logging the kernel does (and systemd, too).  The U-boot setting isn't needed, since you're not likely to be using U-Boot in your builds.

**Comment by slip76** [comment]
@madisongh thanks a lot - your solution perfectly solves my needs. One more question - how can I override this setting in my distro not to touch anything in "meta-mender"?

**Comment by madisongh** [comment]
Adding the following to your distro configuration file should take care of it:
```
KERNEL_ARGS_remove_jetson-xavier = "console=tty0"
KERNEL_ARGS_append_jetson-xavier = " quiet"
```
